test/FunctionalTests/Tests/DataServices/UnitTests/Client.TDD.Tests/Tests/Materialization/PrimitivePropertyConverterTests.cs

```diff
@@ -255,13 +255,13 @@ public void AtomConversionForDateTimeShouldThrow()
         {
             Type dateTimeType = typeof(DateTime);
             Action test = () => this.atomConverter.ConvertPrimitiveValue(DateTimeOffset.UtcNow, dateTimeType);
-            test.ShouldThrow<InvalidOperationException>(Microsoft.OData.Client.Strings.ClientType_UnsupportedType(dateTimeType.FullName));
-        }
-
-        [TestMethod]
+            test.ShouldThrow<InvalidOperationException>(Microsoft.OData.Client.Strings.ClientType_UnsupportedType(dateTimeType.FullName));

```

Comment:
@jingjingrong, Have you tested this case, It should not throw exception now, right?